code/pizzabot.js

```diff
@@ -1,7 +1,73 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
 
-//Put your Javscript code here:
+const submitForm = document.getElementById("form-action");
+
+document.getElementById("welcomeMessage").innerHTML=(`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`);
+
+/*A function that checks if userinput-choice is on menu. Returns true or false. The argument 
+is the input from the form in html and is passed as a value to the parameter "chosenPizza" in the js function. 
+Also uses the toLowerCase method to allow for users using capitals and lower case. Stores the true
+value in orderName. Is checked in if-statement: if true a particular code is run. If false, another code is run. ????*/ 
+const validateOrderName = (chosenPizza) => {
+   if (chosenPizza.toLowerCase() === vegetarian.toLowerCase() || chosenPizza.toLowerCase() === pepperoni.toLowerCase() || chosenPizza.toLowerCase() === hawaiian.toLowerCase()) {
```

Comment:
Great validation! I'm even more happy that you apply method `.toLowerCase()` both to variable and parameter passed from user. It's not so obvious, someone else would add 6 conditionals - 3x per lowerCase and 3x per "normal case". In your approach, you already do `toLowerCase()` on both stored variable and user text, so we don't have to worry about differences.

You can add one more thing to achieve superior clean code so to speak :

create local variables (so, inside `validateOrderName ` function) where you will store `vegetarian.toLowerCase()` and two others. Thanks to that, inside if statement you will not have to use `toLowerCase()` method and whole line will be shorter. That's clean code requirement. From mechanic point of view, you couldn't do it better that it already is :)